blockchain/blockdao/blockdao.go

```diff
@@ -164,15 +164,23 @@ func (dao *blockDAO) checkIndexers(ctx context.Context) error {
 			if err != nil {
 				return err
 			}
-			if err := indexer.PutBlock(protocol.WithBlockCtx(
-				ctxWithTip,
-				protocol.BlockCtx{
-					BlockHeight:    i,
-					BlockTimeStamp: blk.Timestamp(),
-					Producer:       producer,
-					GasLimit:       g.BlockGasLimit,
-				},
-			), blk); err != nil {
+			for retry := 1; retry <= 10; retry++ {
```

Comment:
is this b/c of the unsorted map bug?

Reply:
This change is added by accident. It has been added into another PR. I will revert this.